### PR TITLE
Return variable if not empty

### PR DESCRIPTION
**Pull request by CPWstatic**
The **return** sentence is used for returning variable if it is not empty.

We have CN-docs with pr #1246

### CODE REVIEW

**Comment by CPWstatic** [comment]
Jenkins go!

File: src/graph/ReturnExecutor.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            LOG(ERROR) << "Get rows failed: " << ret.status();
+            onError_(std::move(ret).status());
+            return;
+        }
+        auto rows = ret.value();
+        if (rows.empty()) {
+            onFinish_(Executor::ProcessControl::kNext);
+            return;
+        }
+        resp_->set_rows(std::move(ret).value());
```
Comment (monadbobo) [inline src/graph/ReturnExecutor.cpp]: std::move(rows)

File: src/parser/ProcessControlSentences.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+#include "parser/ProcessControlSentences.h"
+
+namespace nebula {
+std::string ReturnSentence::toString() const {
+    std::string buf;
+    buf.reserve(256);
+
+    buf += "RETURN $";
```
Comment (laura-ding) [inline src/parser/ProcessControlSentences.cpp]: if var_ is nullptr, here  has not ”RETURN $"

File: src/parser/parser.yy [hunk 3]
```diff
@@ -1721,13 +1723,26 @@ maintain_sentence
     | balance_sentence { $$ = $1; }
     ;
 
+return_sentence
+    : KW_RETURN VARIABLE {
```
Comment (laura-ding) [inline src/parser/parser.yy]: why?

File: src/parser/ProcessControlSentences.cpp (hunk 2 above)
Comment (CPWstatic) [reply src/parser/ProcessControlSentences.cpp]: Actually the var_ shall never be nullptr, or fatal error might occurred somewhere.

**Comment by zhangguoqing** [comment]
LGTM!

**Comment by jude-zhu** [comment]
relate to #174

**Pull request merged**